ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-isis.j2

```diff
@@ -36,6 +36,29 @@
 {%     if router_isis.segment_routing_mpls.enabled is arista.avd.defined %}
 | SR MPLS Enabled | {{ router_isis.segment_routing_mpls.enabled }} |
 {%     endif %}
+{%     if router_isis.redistribute_routes is arista.avd.defined %}
+
+### ISIS Route Redistribution
+
+| Route Type | Route-Map | Include Leaked |
+| ---------- | --------- | -------------- |
+{%         for redistribute_route in router_isis.redistribute_routes | arista.avd.convert_dicts('source_protocol') | arista.avd.natural_sort('source_protocol') %}
```

Comment:
`convert_dicts` is not needed, since this is a new data model. We only use that filter to automatically convert old data models to new.
```suggestion
{%         for redistribute_route in router_isis.redistribute_routes | arista.avd.natural_sort('source_protocol') %}
```